memoapp.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+
+MEMOS_FILE = 'public/memos.json'
+
+before do
+  @memo = JSON.parse(File.read(MEMOS_FILE))
+end
+
+get '/memos' do
+  @page_title = 'Top'
+  erb :home, layout: :layout
```

Comment:
https://sinatrarb.com/intro.html#views--templates に `This will render views/index.erb embedded in the views/post.erb (default is views/layout.erb, if it exists).`  と書いている通り、`views/layout.erb` にlayoutをおいている限り、デフォルトで読み込まれます。